README.md

```diff
@@ -1,18 +1,15 @@
-# Atutomatio_training
-automation training
+#Automation_training 
+For test "modifiedBoard.cy.js"
```

Comment:
You'll have conflicts modifying this text in different Pull Requests, I suggest you just remove it.